packages/package_genometools_1_5_7/tool_dependencies.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<tool_dependency>
```

Comment:
I think we should move all packages into IUC. This one con probably be used by other than the RNA community, right?